packages/fx-core/src/plugins/resource/simpleauth/utils/common.ts

```diff
@@ -145,6 +146,17 @@ export class Utils {
     return configValue;
   }
 
+  public static handleError(error: any, errorDetail: PluginError, ...args: string[]): FxError {
+    if (
+      error?.response?.status >= Constants.statusCodeUserError &&
```

Comment:
Do we make sure all the error content structure has response.status? Do we need have type to restrict this?

Reply:
Here handleError method only works on ARM response for creating webapp, update webapp config, etc. It should be better if we have a type for the error. I will update this pr, thx!